src/api/test/functional/search_controller_test.rb

```diff
@@ -425,7 +429,7 @@ def test_find_owner
 
     get "/search/owner?binary='package'"
     assert_response :bad_request
-    assert_xml_tag tag: 'status', attributes: { code: '400', origin: 'backend' }
+    assert_xml_tag tag: 'status', attributes: { code: 'attribute_not_set' }
```

Comment:
Originally this is not testing that the attribute is not present, but that the syntax is wrong.

```
I, [2025-04-25T10:28:53.999947 #11855]  INFO -- : 20250425T102853 POST backend:5352/search/published/binary/id?match=(@name=''package''+and+(@project='Apache')) 400 0.061340555
I, [2025-04-25T10:28:54.000820 #11855]  INFO -- : <status code="400">
  <summary>missing ) in expression</summary>
</status>
```

Reply:
Let's just drop 0c19104393dfd68756297b1cee94b4c1d6789a75 from this PR please. Its not fixing anything.

Reply:
0c19104 gives proper feedback if the user specifies a not existing attribute on the api request. It helps a lot when you debug some complex setup.

Reply:
Please read this comment: 

https://github.com/openSUSE/open-build-service/pull/9964?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDE2MDQ2ODIwNjYzOjUxNDc4NQ%3D%3D&notifications_query=repo%3AopenSUSE%2Fopen-build-service#discussion_r2059989927